src/react/components/pages/editorPage/editorSideBar.tsx

```diff
@@ -21,6 +21,7 @@ export interface IEditorSideBarProps {
  */
 export interface IEditorSideBarState {
     selectedAsset: IAsset;
+    scrollToIndex: number;
```

Comment:
I would suggest pulling this change to a different PR since it's fixing a different issues